src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = days * 40;
+
+  if (days >= 7) {
+    return basePrice - 50;
```

Comment:
Use a constant for the number of days that qualifies for a long term rental and the corresponding discount amount, instead of hardcoding the values `7` and `50`.